packages/core/src/util/initializedTooLateHeuristic.js

```diff
@@ -20,7 +20,7 @@ let patterns = [
   /\/@elastic\/elasticsearch\/index.js/,
   /\/@google-cloud\/pubsub\/build\/src\/publisher\/index.js/,
   /\/@google-cloud\/pubsub\/build\/src\/subscriber.js/,
-  /\/@google-cloud\/storage\/build\/src\/index.js/,
+  /\/@google-cloud\/storage\/build\/cjs\/src\/index.js/,
```

Comment:
Maybe we should keep both entries, the old one and the new path, so that the detection works for older as well as for newer package versions?

Reply:
Pushed the changes as per review.